modules/@apostrophecms/piece-type/ui/apos/components/AposDocsManager.vue

```diff
@@ -247,7 +247,6 @@ export default {
       });
       if (!doc) {
         // Cancel clicked
-        return;
```

Comment:
It looks to me like assigning to doc and having this if statement at all is pointless now, no? I imagine at one point there must have been more logic in the method for the case where doc exists, but there is none now. Probably because we do whatever it was in an event listener instead.

Reply:
Ah, you are correct. I'll excise more.

Reply:
Yeah it used to continue to relationship stuff https://github.com/apostrophecms/apostrophe/blame/c72f42b0ea209de5400ea0b405b8a8e841a3c9ea/modules/@apostrophecms/piece-type/ui/apos/components/AposDocsManager.vue#L345-L352